boa3/model/builtin/classmethod/lowermethod.py

```diff
@@ -0,0 +1,158 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+from boa3.model.builtin.method.builtinmethod import IBuiltinMethod
+from boa3.model.type.primitive.bytestype import BytesType
+from boa3.model.type.primitive.strtype import StrType
+from boa3.model.variable import Variable
+from boa3.neo.vm.opcode.Opcode import Opcode
+
+
+class LowerMethod(IBuiltinMethod):
+    def __init__(self, self_type: Union[StrType, BytesType] = None):
+        from boa3.model.type.type import Type
+
+        if not isinstance(self_type, (StrType, BytesType)):
+            self_type = Type.str
+
+        identifier = 'lower'
+        args: Dict[str, Variable] = {'self': Variable(self_type)}
+
+        super().__init__(identifier, args, return_type=self_type)
+
+    @property
+    def _arg_self(self) -> Variable:
+        return self.args['self']
+
+    @property
+    def opcode(self) -> List[Tuple[Opcode, bytes]]:
+        from boa3.compiler.codegenerator import get_bytes_count
+        from boa3.neo.vm.type.StackItem import StackItemType
+        from boa3.neo.vm.type.Integer import Integer
+
+        upper_a = Integer(65).to_byte_array()
+        upper_z = Integer(90).to_byte_array()
```

Comment:
```suggestion
        upper_a = Integer(ord('A')).to_byte_array()
        upper_z = Integer(ord('Z')).to_byte_array()
```
easier to understand than the magic numbers